compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -529,9 +529,14 @@ void buildAMDAIETransformPassPipeline(OpPassManager &variantPassManager) {
   }
   modulePassManager.addPass(createLowerUKernelOpsToCallsPass());
   if (clUsePipeline == AIEPassPipeline::PadPackPipeline) {
-    addMLIRAIRLoweringPasses(modulePassManager, false);
+    addMLIRAIRLoweringPasses(modulePassManager,
+                             AIEPassPipeline::PadPackPipeline);
   } else if (clUsePipeline == AIEPassPipeline::PackPeelPipeline) {
-    addMLIRAIRLoweringPasses(modulePassManager, true);
+    addMLIRAIRLoweringPasses(modulePassManager,
+                             AIEPassPipeline::PackPeelPipeline);
+  } else if (clUsePipeline == AIEPassPipeline::ConvDecomposePipeline) {
+    addMLIRAIRLoweringPasses(modulePassManager,
+                             AIEPassPipeline::ConvDecomposePipeline);
```

Comment:
I don't think you need to pass AIEPassPipeline as a variable. Instead, you can call `addMLIRAIRLoweringPasses(modulePassManager)` directly and use `clUsePipeline` inside the function directly.

Reply:
Make sense. Updated.